contracts/mainnet/connectors/notional/helpers.sol

```diff
@@ -0,0 +1,286 @@
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+import {Token, NotionalInterface, BalanceAction, BalanceActionWithTrades, DepositActionType} from "./interface.sol";
+import {Basic} from "../../common/basic.sol";
+import {DSMath} from "../../common/math.sol";
+import {TokenInterface} from "../../common/interfaces.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    uint8 internal constant LEND_TRADE = 0;
+    uint8 internal constant BORROW_TRADE = 1;
+    uint256 internal constant INTERNAL_TOKEN_PRECISION = 1e8;
+    uint256 internal constant ETH_CURRENCY_ID = 1;
+    uint256 internal constant MAX_DEPOSIT = uint256(-1);
+
+    /// @dev Contract address is different on Kovan: 0x0EAE7BAdEF8f95De91fDDb74a89A786cF891Eb0e
+    NotionalInterface internal constant notional =
+        NotionalInterface(0x1344A36A1B56144C3Bc62E7757377D288fDE0369);
+
+    /// @notice Returns the address of the underlying token for a given currency id,
+    function getUnderlyingToken(uint16 currencyId)
+        internal
+        view
+        returns (address)
+    {
+        // prettier-ignore
+        (/* assetToken */, Token memory underlyingToken) = notional.getCurrency(currencyId);
+        return underlyingToken.tokenAddress;
+    }
+
+    /// @notice Returns the address of the asset token for a given currency id
+    function getAssetToken(uint16 currencyId) internal view returns (address) {
+        // prettier-ignore
+        (Token memory assetToken, /* underlyingToken */) = notional.getCurrency(currencyId);
+        return assetToken.tokenAddress;
+    }
+
+    function getCashBalance(uint16 currencyId)
+        internal
+        view
+        returns (int256 cashBalance)
+    {
+        // prettier-ignore
+        (
+            cashBalance,
+            /* int256 nTokenBalance */,
+            /* int256 lastClaimTime */
+        ) = notional.getAccountBalance(currencyId, address(this));
+    }
+
+    function getNTokenBalance(uint16 currencyId)
+        internal
+        view
+        returns (uint256)
+    {
+        // prettier-ignore
+        (
+            /* int256 cashBalance */,
+            int256 nTokenBalance,
+            /* int256 lastClaimTime */
+        ) = notional.getAccountBalance(currencyId, address(this));
+        return toUint(nTokenBalance);
+    }
```

Comment:
These both too, as they perform the same actions but returns different thing

Reply:
done